tools/check.py

```diff
@@ -667,6 +667,42 @@ def command_line():
 
     return parser.parse_args()
 
+def check_required_files(files):
+    """Check if required files exists."""
+    REQUIRED_FILES = ["..-.*\.md",
```

Comment:
If regex is to be used to enforce the requirement of one lesson file, then `..-` is too broad, as we always expect the part before the hyphen to be solely numeric.

Realistically, we're checking for the existence of 1 or more lesson files, so "01-*.md" in glob would probably be sufficient. (validating `02-lesson.md` without lesson 1 would be odd)